election/serializers.py

```diff
@@ -89,9 +89,19 @@ def __init__(self, name, idx, profile, grade):
         self.profile = profile
         self.grade = grade
 
-
 class CandidateSerializer(serializers.Serializer):
     name = serializers.CharField()
     id = serializers.IntegerField(min_value=0)
     profile = serializers.DictField(child=serializers.IntegerField())
     grade = serializers.IntegerField(min_value=0, max_value=settings.MAX_NUM_GRADES)
+    num_votes = serializers.IntegerField(min_value=0)
+
+class LinkSerializer(serializers.Serializer):
+
+    id_election=serializers.CharField()

```

Comment:
In the project we use the convention `election_id` instead of `id_election`. I think it's a convention globally used in Django.